src/Domain/DonationService.php

```diff
@@ -322,7 +312,6 @@ public function confirmOnSessionDonation(
             $paymentIntentId,
             [
                 'confirmation_token' => $tokenId->stripeConfirmationTokenId,
-                ...self::ASYNC_CAPTURE_OPT_OUT
```

Comment:
so its only three months but I can't immediately remember why I added this here in https://github.com/thebiggive/matchbot/pull/1183/files#diff-bd27501321f38ab1ad31b1e5c4640c8f32b128dd8b282103ad0a6801476c87f8R69 . Trying to work that out to check if I misunderstood something then or if there's a reason we should keep it.